commands/add-language.ts

```diff
@@ -56,25 +62,37 @@ export default class AddLanguageCommand extends BaseCommand {
   #generateDeterministicUUID(courseSlug: string, languageSlug: string, suffix: string): string {
     const input = `${suffix}-${courseSlug}-${languageSlug}`;
     let hash = 0;
-    
+
     // Generate a simple hash from the input string
     for (let i = 0; i < input.length; i++) {
       const char = input.charCodeAt(i);
-      hash = ((hash << 5) - hash) + char;
+      hash = (hash << 5) - hash + char;
       hash = hash & hash; // Convert to 32-bit integer
     }
 
     // Create multiple hash values for different segments
-    const hash1 = Math.abs(hash).toString(16).padStart(8, '0').slice(0, 8);
-    const hash2 = Math.abs(hash * 31).toString(16).padStart(4, '0').slice(0, 4);
-    const hash3 = Math.abs(hash * 47).toString(16).padStart(4, '0').slice(0, 4);
-    const hash4 = Math.abs(hash * 67).toString(16).padStart(4, '0').slice(0, 4);
-    const hash5 = Math.abs(hash * 97).toString(16).padStart(12, '0').slice(0, 12);
+    const hash1 = Math.abs(hash).toString(16).padStart(8, "0").slice(0, 8);
+    const hash2 = Math.abs(hash * 31)
+      .toString(16)
+      .padStart(4, "0")
+      .slice(0, 4);
+    const hash3 = Math.abs(hash * 47)
+      .toString(16)
+      .padStart(4, "0")
+      .slice(0, 4);
```

Comment:
@andy1li the line length here looked fine. We have this configured in package.json for prettier to pick up. Mind checking your IDE config?

Reply:
https://github.com/codecrafters-io/course-sdk/blob/bf9230395bfdde1553d740b5f67f20dfcbe4c457/package.json#L31